.github/workflows/test.yml

```diff
@@ -40,3 +40,9 @@ jobs:
         run: |
           export SIMPLIFIED_STATIC_DIR="$(pwd)/resources"
           tox
+
+      - name: Coverage HTML Report
+        uses: actions/upload-artifact@v3
```

Comment:
The default retention period is 90 days which I think is longer than it needs to be for these artifacts. I'm not sure what a good retention period would be, maybe 14 or 21 days?

Reply:
14 days is good since it guarantees the artifact is available for the rest of the current sprint. 30 days would also be fine. I agree that 90 is excessive.